nltk/test/unit/lm/test_preprocessing.py

```diff
@@ -13,16 +13,16 @@ class TestPreprocessing(unittest.TestCase):
     def test_padded_everygram_pipeline(self):
         expected_train = [
             [
-                ("<s>",),
-                ("a",),
-                ("b",),
-                ("c",),
-                ("</s>",),
-                ("<s>", "a"),
-                ("a", "b"),
-                ("b", "c"),
-                ("c", "</s>"),
-            ]
+                 ('<s>',),
```

Comment:
Hmm, how did this change happen? The automatic formatter we use (`black`) enforces both double quotes and commas at the end of lists.

Did you get a chance to run it?

Reply:
Thanks for pointing this out. I only ran `tox -e py38` as directed by [CONTRIBUTING.md](https://github.com/nltk/nltk/blob/develop/CONTRIBUTING.md). I am currently getting an `InvocationError` when running `tox -e py-travis` and need to look into this. I can revert the formatting here though!

Reply:
Hmm, indeed the automatic formatting hasn't made it into the guidelines, sorry about that. @alvations was there any interest in setting up pre-commit hooks to automate this?

Reply:
Yes but I didn't manage to get the black to work pre-commit. In any case the examples in the test cases shouldn't change convention, i.e. doubles quotes and black formatted.